weeklymission2/src/useHooks/useFiles.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from "react";
+import { getFolderName, getLinkData } from "../APIs/FolderPageApi";
+
+export const useFiles = () => {
+  const [folder, setFolder] = useState([]);
+
+  const fetchFile = async () => {
+    const { data } = await getFolderName();
+    setFolder(data);
+  };
+
+  useEffect(() => {
+    fetchFile();
+  }, []);
+
+  return folder;
```

Comment:
훅을 리턴값에 대해서는 값이 하나라도 객체로 리턴하는게 좋아요
```suggestion
  return {folder};
```